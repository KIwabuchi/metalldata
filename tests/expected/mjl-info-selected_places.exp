{"returns":[{"rank":0,"elements":16,"selected":11},{"rank":1,"elements":0,"selected":0},{"rank":3,"elements":0,"selected":0},{"rank":2,"elements":0,"selected":0}]}
