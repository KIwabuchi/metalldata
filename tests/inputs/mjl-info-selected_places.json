{"_state": {"metall_location": "/PATH/TO/DATASTORE/m_places", "selected": [{"expression_type": "jsonlogic", "rule": {"and": [{">=": [{"var": "keys.place"}, "Elk"]}, {"<": [{"var": "keys.id"}, 100]}]}}]}}
