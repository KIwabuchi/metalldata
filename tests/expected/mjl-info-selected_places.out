{"returns":[{"rank":0,"elements":0,"selected":0}]}
